### ISSUE TITLE
oasisMRI/A structure  and oasis brain data

### ISSUE THREAD

**Issue by alqurri77** [csbbbv/ColorMedGAN#3]
Hi;

I download the oasis data from [https://www.oasis-brains.org/#data], I notice there are so many files and folder, for example,  there are:
 'PROCESSED' and 'RAW' directory and there are many files under those directories, but **data_load** funcation require a path to and  subfolder. So I'm confused to which directory should the path point to and which  subfolder.

`train_loader_src = utils.data_load(os.path.join('./data', args.src_data), 'train', transform, args.batch_size, shuffle=True, drop_last=True)
`
```
def data_load(path, subfolder, transform, batch_size, shuffle=False, drop_last=True):
    dset = datasets.ImageFolder(path, transform)
    ind = dset.class_to_idx[subfolder]

    n = 0
    for i in range(dset.__len__()):
        if ind != dset.imgs[n][1]:
            del dset.imgs[n]
            n -= 1
        n += 1

    return torch.utils.data.DataLoader(dset, batch_size=batch_size, shuffle=shuffle, drop_last=drop_last)

```

**Comment by TardisBlue2** [comment]
The structure is like:
├── datasets                  
|   ├── src_data         # gray
|   |   ├── train
|   |   ├── test 
|   ├── tgt_data         # color
|   |   ├── train  
|   |   ├── test 

You can refer to the organization under this link：https://github.com/YuSuen/colorCycleGAN

**Comment by alqurri77** [comment]
Hi;

Thank you. I'm not sure what files to place from oasis-brains dataset. Because when I downloaded oasis-brains datasets. I got below structure (see image below):

![image](https://user-images.githubusercontent.com/75499343/226483864-ca2d7407-cad7-4bc8-8daa-b4312e0eeafb.png)

So as you can see under "RAW" directory there are many files. which of those to use. 
Also, there is "PROCESSED" directory.

![image](https://user-images.githubusercontent.com/75499343/226484025-3837b502-9b13-40e4-ad8b-53fe1fc14c08.png)
 
Which again contains many files.

**Comment by TardisBlue2** [comment]
It seems that the data format provided by the official website is different from when I downloaded it, in short, you should convert the 3D raw data and labels into a slice-by-slice 2D PNG/JPG format for training UNet and CycleGAN
the folder is like:
![image](https://user-images.githubusercontent.com/48145504/226596699-e04305d8-7671-4a34-a84a-5718b4a2d5bf.png)

**Comment by alqurri77** [comment]
Do you by any chance have the link where you downloaded the original data?
I would rather use the same dataset format, just in case it might lead to different result if I processed the data.

**Comment by alqurri77** [comment]
Also, "tgt_data" should have colored images, which means 3 channels. However, if I took a slice, it will be only gray image.

**Comment by TardisBlue2** [comment]
> 

I didn't do any preprocessing, I just converted the source data into the format of the slice and read it.

**Comment by TardisBlue2** [comment]
> Also, "tgt_data" should have colored images, which means 3 channels. However, if I took a slice, it will be only gray image.

The tgt_data data is originally an RGB image from the Visible Human project, If you need more details, you can take a look at the paper
